include/lsst/jointcal/ConstrainedPhotometryModel.h

```diff
@@ -103,8 +103,9 @@ class ConstrainedFluxModel : public ConstrainedPhotometryModel {
 public:
     explicit ConstrainedFluxModel(CcdImageList const &ccdImageList, afw::geom::Box2D const &focalPlaneBBox,
                                   int visitOrder = 7, double errorPedestal_ = 0)
-            : ConstrainedPhotometryModel(ccdImageList, focalPlaneBBox, visitOrder, errorPedestal_) {
-        _log = LOG_GET("jointcal.ConstrainedFluxModel");
+            : ConstrainedPhotometryModel(ccdImageList, focalPlaneBBox,
+                                         LOG_GET("jointcal.ConstrainedFluxModel"), visitOrder,
+                                         errorPedestal_) {
```

Comment:
Yes, this is exactly what I had in mind... though I underestimated the number of classes whose constructor signatures would need to be changed. 😞